next.config.js

```diff
@@ -3,8 +3,19 @@ const nextConfig = {
     reactStrictMode: true,
     swcMinify: true,
     eslint: {
-        dirs: ['pages', 'components', 'api', 'common']
-    }
+        dirs: ['pages', 'components', 'api', 'common'],
+    },
+    images: {
+        remotePatterns: [
+            {
+                protocol: 'https',
+                hostname: 'okn-mk.mkrf.ru',
```

Comment:
Is it still required for `<Image>` with custom loader?

Reply:
Totally forgot about it! Will check this later

Reply:
Removed